docs/users-guide/05-visualizing-results.md

```diff
@@ -148,7 +148,9 @@ This auto-pivoting is distinct from the pivot table visualization, which we cove
 
 Pivot tables allow you swap rows and columns, group data, and include subtotals in your table. You can group one or more metrics by one or more dimensions. 
 
-For a given query result, you can assign fields to one of three "buckets":
+Pivot tables are only available to simple and custom questions with summarized data. They don't work for questions that lack aggregate data, and they don't work for questions written in SQL, as Metabase would need to modify your SQL code in order to calculate subtotals. If you really need to use SQL, the workaround here is to create your question in two steps: first do all the complex things you need to do in SQL, save the results as a question, then use that saved SQL question as the starting point for a new GUI question which summarizes that data. See [Referencing saved questions in queries](referencing-saved-questions-in-queries.md).
```

Comment:
> See [Referencing saved questions in queries](referencing-saved-questions-in-queries.md)

Sorry, I meant the article about selecting a saved question as starting data in the _notebook_, not using the CTE syntax in SQL. Because what we're telling them to do here is used a saved sql query's results as starting data for a notebook question that they can then visualize as a pivot table.